common/schemas/apdNew.js

```diff
@@ -1,6 +1,19 @@
 import Joi from 'joi';
+import { medicaidBusinessAreasSchema } from './apdOverview.js';
```

Comment:
is there any way to combine this schema and the APD overview schema? Maybe returning a subset to use in the apdNew page?